src/dosbox.cpp

```diff
@@ -1140,6 +1140,10 @@ void DOSBOX_Init() {
 	               "If set to 0, the country code corresponding to the\n"
 	               "selected keyboard layout will be used.");
 
+	Pbool = secprop->Add_bool("expand_shell_variable", when_idle, false);
+	Pbool->Set_help("Enable expanding environment variables such as %PATH%\n"
```

Comment:
I suggest to extend the help string, it should inform the user which real DOS versions supports expanding environment variables.

Reply:
@FeralChild64 Thanks for the feedback. It appears that all versions of FreeCOM and MS-DOS 7/8 COMMAND supports this behavior. Extended the help string as suggested.